### ISSUE TITLE
Phase Carbine sets itself to wielded when it cannot be wielded.

### ISSUE THREAD

**Issue by MisterGrimm** [CHOMPStation2/CHOMPStation2#54]
#### Brief description of the issue
Title says most of it, if a phase carbine is put in a suit storage slot or a belt slot it will sometimes set itself to wielded, when that weapon cannot be wielded with two hands.

#### What you expected to happen
- Place Phase Carbine in suit storage/belt.
- Name stays the same.

#### What actually happened
- Phase Carbine is placed in suit storage/belt.
- Name changes to "Phase Carbine (Wielded)".

#### Steps to reproduce
- Place Phase Carbine in various slots it can fit in.

**Comment by Razgriz1032** [comment]
Does this have an impact on gameplay?

**Comment by MisterGrimm** [comment]
From my experience, no. It is purely a annoying little name change. If there is any mechanic impacts I have not noticed them.

**Comment by ExabyteDiscord** [comment]
After a little bit of digging, found out that this item does have a penalty for not two handing it. Since this can affect game balance, marking as Medium Priority.

**Comment by Darlantanis** [comment]
Is this still a problem?

**Comment by MisterGrimm** [comment]
Probably, although I think I misunderstood how wielding worked when I first made this issue. I figured it was like current wielding code in other codebases, which require you to manually wield it for two handed, not like the weird system we have at the moment where it automatically wields based on checks. 

Although, there is still the small bug of how some two handed wielded items permanently add the "Wielded" name to a item if you put it away in specific ways. Probably has something to do with the automatic checks getting bypassed.
